browser/resources/settings/ad_block_only_mode_page/ad_block_only_mode_page.html

```diff
@@ -0,0 +1,25 @@
+<style include="settings-shared md-select">
```

Comment:
nit: md-select styles are unused (we don't have a `<select>` on the page
```suggestion
<style include="settings-shared">
```